.github/workflows/tests.yaml

```diff
@@ -537,7 +537,7 @@ jobs:
     steps:
       - name: 📂 Checkout code
         uses: actions/checkout@v4
-      - run: rustup override set nightly-2024-06-30
+      - run: rustup override set nightly-2025-01-12
```

Comment:
I updated this line as it seemed to exist for the past several years, but I don't understand the value of its existence.

@max-sixty

Reply:
Great! This was just to lock a version so it didn't randomly fail. Thanks for updating!